comtypes/tools/codegenerator.py

```diff
@@ -345,6 +346,11 @@ def ArrayType(self, tp):
     _enumvalues = 0
     def EnumValue(self, tp):
         value = int(tp.value)
+        if keyword.iskeyword(tp.name):
+            # XXX use logging!
+            if __debug__:
```

Comment:
No, there wasn't any logging in this file but there is in other parts of the library.

I don't like the idea of simply commenting out the print statement, because if/when someone does have a COM object that uses a Python keyword other than `True` or `False` they are not going to know to look for this.

The pattern I was following was the use of the `__verbose__` flag in `client._generate.py`. I like this precedent because it only flags and prints the code generation. Note that `client._generate.py` also  uses `logging` but doesn't for those messages.

Perhaps if I followed that pattern properly (adding the flag at the top), but used a name that better expresses my intentions...